specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md

```diff
@@ -0,0 +1,247 @@
+|     GUID_POWER_SAVING_STATUS             |     Battery   saver has been turned on or off.                                                                                                                                                                                       |     Equivalent   to PowerManager. EnergySaverStatusChanged, EnergySaverStatus.                           |
+|     GUID_POWERSCHEME_PERSONALITY         |     The   active power scheme personality has changed. All power schemes map to one of   these personalities.                                                                                                                        |     PowerSchemePersonalityChanged,   PowerSchemePersonality { HighPerformance, PowerSaver, Balanced}.    |
+|     GUID_SESSION_DISPLAY_STATUS          |     The   display associated with the application's session has been powered on or off.                                                                                                                                              |     Covered   by GUID_CONSOLE_DISPLAY_STATE.                                                             |
+|     GUID_SESSION_USER_PRESENCE           |     The   user status associated with the application's session has changed.                                                                                                                                                         |     UserPresenceStatusChanged,   UserPresenceStatus {Present, Absent}.                                   |
+|     GUID_SYSTEM_AWAYMODE                 |     The   system is entering or exiting away mode.                                                                                                                                                                                   |     SystemAwayModeStatusChanged,   SystemAwayModeStatus {Entering, Exiting}.                             |
+
+The existing PowerManager class exposes only static properties and events: all members are valid to be brought over to the Reunion version. The top-level classes and enums are listed in the following table:
+
+|     New Classes and Enums                                                    |     Description                                                               |
+|------------------------------------------------------------------------------|-------------------------------------------------------------------------------|
+|     enum   BatteryStatus {NotPresent, Discharging, Idle, Charging}           |     Clone   of Windows.System.Power.BatteryStatus.                            |
```

Comment:
Some of these ended up with extra spacing between `Clone` and `of` for instance. Not a huge deal, but in the new names column  you'll want to shorten to `enum BatteryStatus` for instance

Reply:
Fixed.